app/src/main/res/layout/activity_profile.xml

```diff
@@ -240,4 +240,14 @@
         app:layout_constraintTop_toTopOf="parent"
         tools:text="Joe Biden's Portfolio" />
 
+    <ProgressBar
+        android:id="@+id/pbApi"
```

Comment:
Good work with renaming your variables to something easier to read and understand the purpose of. But let's rename this to something like "loading_for_incoming_data"